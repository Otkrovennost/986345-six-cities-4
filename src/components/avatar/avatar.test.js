import React from "react";
import renderer from "react-test-renderer";
import Avatar from "./avatar.jsx";

const email = `abc@mail.ru`;

it(`Should Avatar render correctly`, () => {
  const tree = renderer
    .create(<Avatar
      email={email}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
