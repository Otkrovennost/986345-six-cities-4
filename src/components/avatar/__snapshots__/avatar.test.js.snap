// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Avatar render correctly 1`] = `
Array [
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />,
  <span
    className="header__user-name user__name"
  >
    abc@mail.ru
  </span>,
]
`;
